Content/package.json

```diff
@@ -24,5 +30,13 @@
         "webpack-cli": "^3.1.2",
         "webpack-dev-server": "^3.1.14",
         "whatwg-fetch": "^3.0.0"
+    //#if (layout != "none")
+    },
+    "dependencies": {
```

Comment:
do these need to be in dependencies, or can we move them to devDependencies?

Reply:
I don't know, I'm sorry, but I can try if it works with devDependencies when I return home from work.